hathor/transaction/storage/transaction_storage.py

```diff
@@ -677,6 +729,8 @@ def _build_indexes_manager(self) -> IndexesManager:
     def _reset_cache(self) -> None:
         """Reset all caches. This function should not be called unless you know what you are doing."""
         assert self.with_index, 'Cannot reset cache because it has not been enabled.'
+
+        # XXX: these fields would be better of being migrated to the IndexesManager, and probably as proper indexes
```

Comment:
I agree! Let's create an issue to get it done.

Reply:
Done: #415